### ISSUE TITLE
Stored Object: paper:arxiv.1805.10694

### ISSUE THREAD

**Issue by dmarx** [dmarx/papers-feed-dev#6]
{
  "primary_id": "arxiv.1805.10694",
  "source": "arxiv",
  "sourceId": "1805.10694",
  "url": "https://arxiv.org/abs/1805.10694",
  "title": "Exponential convergence rates for Batch Normalization: The power of\n  length-direction decoupling in non-convex optimization",
  "authors": "Jonas Kohler, Hadi Daneshmand, Aurelien Lucchi, Ming Zhou, Klaus Neymeyr, Thomas Hofmann",
  "abstract": "Normalization techniques such as Batch Normalization have been applied\nsuccessfully for training deep neural networks. Yet, despite its apparent\nempirical benefits, the reasons behind the success of Batch Normalization are\nmostly hypothetical. We here aim to provide a more thorough theoretical\nunderstanding from a classical optimization perspective. Our main contribution\ntowards this goal is the identification of various problem instances in the\nrealm of machine learning where % -- under certain assumptions-- Batch\nNormalization can provably accelerate optimization. We argue that this\nacceleration is due to the fact that Batch Normalization splits the\noptimization task into optimizing length and direction of the parameters\nseparately. This allows gradient-based methods to leverage a favourable global\nstructure in the loss landscape that we prove to exist in Learning Halfspace\nproblems and neural network training with Gaussian inputs. We thereby turn\nBatch Normalization from an effective practical heuristic into a provably\nconverging algorithm for these settings. Furthermore, we substantiate our\nanalysis with empirical evidence that suggests the validity of our theoretical\nresults in a broader context.",
  "timestamp": "2025-03-05T07:26:53.574Z",
  "rating": "novote",
  "arxivId": "1805.10694",
  "arxiv_tags": [
    "stat.ML",
    "cs.LG"
  ],
  "published_date": "2018-05-27T21:33:37Z"
}

---
*Copied from dmarx/papers-feed#1550*

**Comment by dmarx** [comment]
{
  "_data": {
    "primary_id": "arxiv.1805.10694",
    "source": "arxiv",
    "sourceId": "1805.10694",
    "url": "https://arxiv.org/abs/1805.10694",
    "title": "Exponential convergence rates for Batch Normalization: The power of\n  length-direction decoupling in non-convex optimization",
    "authors": "Jonas Kohler, Hadi Daneshmand, Aurelien Lucchi, Ming Zhou, Klaus Neymeyr, Thomas Hofmann",
    "abstract": "Normalization techniques such as Batch Normalization have been applied\nsuccessfully for training deep neural networks. Yet, despite its apparent\nempirical benefits, the reasons behind the success of Batch Normalization are\nmostly hypothetical. We here aim to provide a more thorough theoretical\nunderstanding from a classical optimization perspective. Our main contribution\ntowards this goal is the identification of various problem instances in the\nrealm of machine learning where % -- under certain assumptions-- Batch\nNormalization can provably accelerate optimization. We argue that this\nacceleration is due to the fact that Batch Normalization splits the\noptimization task into optimizing length and direction of the parameters\nseparately. This allows gradient-based methods to leverage a favourable global\nstructure in the loss landscape that we prove to exist in Learning Halfspace\nproblems and neural network training with Gaussian inputs. We thereby turn\nBatch Normalization from an effective practical heuristic into a provably\nconverging algorithm for these settings. Furthermore, we substantiate our\nanalysis with empirical evidence that suggests the validity of our theoretical\nresults in a broader context.",
    "timestamp": "2025-03-05T07:26:53.574Z",
    "rating": "novote",
    "arxivId": "1805.10694",
    "arxiv_tags": [
      "stat.ML",
      "cs.LG"
    ],
    "published_date": "2018-05-27T21:33:37Z"
  },
  "_meta": {
    "client_version": "0.3.2",
    "timestamp": "2025-03-05T07:26:54.592Z",
    "update_mode": "append"
  },
  "type": "initial_state"
}

---
*Original comment by @dmarx*
